doc/hostname.md

```diff
@@ -0,0 +1,49 @@
+## Hostname in Linux ##
+
+We have several hostnames in linux. Every of them has different usage and different place where to set it up. Some of them can be configured in the YaST's lan module.
+
+## Local system hostname / static hostname (/etc/hostname) ##
+
+You can put a string into /etc/hostname. This string is than used as so called "local system hostname" which is set in the kernel during the boot. This name is used for identification of your local system. This name is typically used e.g. in logs where individual log entries can be prepended by this name. More importantly this name is not related to networking, so putting e.g. FQDN (Fully Qualified Domain Name) has no sense.
+
+Content of this file can be displayed by ```hostname``` command or in systemd world by ```hostnamectl --static```
+
+In YaST you can edit this hostname on Hostname / DNS tab, hostname field. Unlike other ways mentioned above, YaST validates your input.
+
+## Dynamic system hostname(s) (via DHCP) ##
+
+In modern networks is quite common to use DHCP for configuring workstations. If DHCP server supports that you can even configure hostname for its clients. If you want to use hostname from DHCP server on the client, you have to allow it. You have to configure ```DHCLIENT_SET_HOSTNAME``` option on the client. This option can be configured on global level for all interfaces in ```/etc/sysconfig/network/dhcp``` file - this setup is then used as global default for all interfaces without that option explicitly configured. You can also configure this option per interface in the interface's ifcfg file. It is very easy to put the system into wrong state here and you will get unpredictable hostname(s) for your system.
```

Comment:
```suggestion
In modern networks is quite common to use DHCP for configuring workstations. If the DHCP server supports that it can even configure hostname for its clients. If you want to use hostname from DHCP server on the client, you have to allow it. You have to configure ```DHCLIENT_SET_HOSTNAME``` option on the client. This option can be configured on global level for all interfaces in ```/etc/sysconfig/network/dhcp``` file - this setup is then used as global default for all interfaces without that option explicitly configured. You can also configure this option per interface in the interface's ifcfg file. It is very easy to put the system into wrong state here and you will get unpredictable hostname(s) for your system.
```

Reply:
In such a long suggestion, I can't see the difference :(